sklearnex/linear_model/logistic_regression.py

```diff
@@ -289,6 +289,7 @@ def _onedal_fit(self, X, y, sample_weight=None, queue=None):
             if queue is None or queue.sycl_device.is_cpu:
                 return self._onedal_cpu_fit(X, y, sample_weight)
 
+            #  coverage: gpu_start
```

Comment:
What is the purpose of those comments? And why are they added only for LogReg?

Reply:
Good question! We can avoid code coverage analysis for regions between comments ```coverage: gpu_start``` and ```coverage: gpu_stop``` since we do not do any GPU operations in public CI. If someone does a full GPU-only PR they will see a large decrease in code coverage, which we can designate as GPU code.  Do to so, we need to remove the ```#``` from this file: https://github.com/icfaust/scikit-learn-intelex/blob/dev/pytest-cov/.coveragerc#L23

Reply:
Oh, and I did it only for LogisticRegression as an initial thing because the sklearnex code is only GPU-relevant, as CPU is offloaded to daal4py (and would otherwise show very bad code-coverage metrics).

Reply:
Thanks for the explanation.
But it looks like a lot of work to write all those comments properly in the code and maintain them in good quality.
Can there be a better way to exclude GPU-related code from the code coverage analysis?

Reply:
Maybe it is possible to add something like:
```
exclude_also =
    self\._onedal_estimator\..*queue=queue\)
```
to .coveragerc. To exclude all the estimators calls that use SYCL queue.

Reply:
Since we aren't making a hard and fast rule, it could be that we make the first round of analysis without gpu exclusion, the metrics will show regressions with GPU code introduction, but the way GPU code isn't fully standardized in the codebase, so you are right that it would be a long a tedious process.

Reply:
I'll start by removing the secondary excludes from LogisticRegression and elsewhere, and a development ticket to the backlog (let me know what you think @Vika-F ).  I think this will occur after we integrate into coveralls.